kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -76,8 +77,38 @@
                   :disabled="false"
                   :hasIcons="true"
                   :options="overflowTabs"
-                  @select="opt => quizForge.setActiveSection(opt.id)"
-                />
+                  @select="opt => setActiveSection(opt.id)"
+                >
+                  <template #option="{ option }">
+                    <!-- TODO Clean this up by moving it to another component -->
+                    <!-- Maybe not so easy since they're styled differently -->
+                    <KButton
+                      appearance="flat-button"
+                      :primary="activeSection.value.section_id === option.id"
+                      :appearanceOverrides="tabStyles"
+                      class="menu-button"
+                      @click="() => setActiveSection(option.id)"
+                    >
+                      {{ option.label }}
+                    </KButton>
+                    <KIconButton
+                      icon="optionsVertical"
+                      style="position: absolute; right: 0; border-radius: 0!important;"
+                      @click="() => null"
+                    >
+                      <template #menu>
+                        <KDropdownMenu
+                          :primary="false"
+                          :disabled="false"
+                          :hasIcons="true"
+                          :containFocus="false"
+                          :options="sectionOptions"
+                          @select="opt => handleSectionOptionSelect(opt, option.id)"
+                        />
+                      </template>
+                    </KIconButton>
+                  </template>
+                </KDropdownMenu>
```

Comment:
This is an artifact of where I think this branched from in git. My apologies here

We only need to have this part here between the `KIconButton`

```
<template #menu>
  <KDropdownMenu
    :primary="false"
    :disabled="false"
    :hasIcons="true"
    :options="overflowTabs"
    @select="opt => quizForge.setActiveSection(opt.id)"
  />
</template>
```